src/main/java/ro/hiringsystem/controller/JobTypesController.java

```diff
@@ -0,0 +1,18 @@
+package ro.hiringsystem.controller;
+
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+import ro.hiringsystem.model.enums.JobType;
+
+@RestController
+@RequestMapping("/api/v1/jobTypes")
```

Comment:
I do not consider jobTypes should have its own controller, as it could have been very well mapped to "/api/v1/job/types" or "api/v1/job/types/get". Since job types are stored in an enum in the backend, I find it hard to believe you can create other CRUD methods rather than "get" for them.